lib/config_file.rb

```diff
@@ -17,7 +17,9 @@ def add_section(name)
 
   def read_config
     current_section = nil
-    IO.readlines(File.expand_path(@config_file_location)).each_with_index do |line, i|
+    filename = File.expand_path(@config_file_location)
+    return unless File.exist?(filename)
```

Comment:
Hah, very good callout! I've been dragging my .ssh/config file around for over 10 years and never run into this problem. Thank you!